prelude/README.md

```diff
@@ -150,7 +150,7 @@ products # groupBy productCategory # filter isInStock # length
 is equivalent to
 
 ```purescript
-length (groupBy productCategory (filter isInStock (products)))
+length (filter isInStock (groupBy productCategory (products)))
```

Comment:
Oh, I didn't actually do this, it must have been a change made in `prelude.purs` at some point and then the docs hadn't been regenerated...